Documentation/Docker and PostgreSQL.md

```diff
@@ -27,13 +21,15 @@
         * `docker image rm IMAGE_NAME`
     * delete containers
         * `docker container rm CONTAINER_NAME`
+    * [https://www.ictshore.com/data-center/how-to-use-docker/#:~:text=How%20to%20use%20Docker%201%20Installing%20Docker%20Of,5%20Storage%20persistence%20...%206%20The%20Dockerfile%20](url)
+    * `docker ps`
+        * verifies that the database is running
+    * `psql -h localhost -p 5003 -U postgres -d postgres`
```

Comment:
Always good to be familiar with how the tools work on a lower level, esp because we can expect most students to have to deal with the underlying interfaces later on.